packages/ihe-gateway/server/CodeTemplates/XCA Library - Outbound/index.xml

```diff
@@ -1,104 +1,79 @@
 <?xml version="1.0" encoding="UTF-8"?><codeTemplateLibrary version="4.4.2">
-  <id>c209459e-8282-4ac7-88cb-42cdeec33805</id>
-  <name>Common Library</name>
-  <revision>826</revision>
+  <id>5c4c6ab2-26a7-4878-933d-ae04c2bdeea7</id>
+  <name>XCA Library - Outbound</name>
+  <revision>602</revision>
```

Comment:
Any reason not to revert back to the revision (80) here to keep the revert consistent? https://github.com/metriport/metriport/pull/1965/files#diff-3a987f8e55041bf517bccc7bf2d6b407afd4a4249f5aacb9343a8e034c9a8a4b